python_backend/services/profile_generator.py

```
@@ -0,0 +1,38 @@
+        - 주된 소통 성격: {survey_data.communication_style}
+        - 주 소통 채널: {survey_data.main_channel}
+        - 주 커뮤니케이션 대상: {', '.join(survey_data.main_target)}
+
+        이 정보를 바탕으로, 이 회사의 신입사원이 따라야 할 커뮤니케이션 스타일 프로필을
+        핵심 원칙 2~3가지로 요약해서 생성해줘.
+        """
+        
+        print(f"--- LLM에 전달할 프롬프트 ---\n{prompt}")
+        # 실제 OpenAI API 호출
+        profile_text = (
+            f"1. 친근하고 수평적인 소통을 지향하며, 주로 Slack을 통해 신속하게 의견을 교환합니다.\n"
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)